code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -61,19 +63,25 @@
 			playsound(Y,'sound/effects/thunder.ogg', 90, 1)
 			spawn(10)
 				user.gib()
+		if(ishuman(user) || ismonkey(user))

```

Comment:
This is kinda snowflakey. I think it might be best if the default value is fart.ogg and normal blood and then from there we can check if they're alien or something. Otherwise we need to handle every subtype if we add new ones